### PR TITLE
New Pom to aid the build

### PR DESCRIPTION
**Pull request by ghostlypi**
New pom allows the downloading of the most up to date dependencies including steamworks4j. No windows script updates coming due to WSL solving all our windows problems for little to no performance loss

### CODE REVIEW

**Comment by aehmttw** [comment]
Looks like you included a bunch of jars in the commit - are you sure you want to have these here?

**Comment by ghostlypi** [comment]
That should have removed all the Artifacts...

**Comment by aehmttw** [comment]
I think you might have to remove them from the history too, because right now if people would pull this, it would still download all the jars as history and then delete them (leading to large download sizes).

**Comment by ghostlypi** [comment]
Purged the history of all artifacts. Should be good now.

File: run.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+./jdk-17.0.2.jdk/Contents/Home/bin/java -XstartOnFirstThread -jar Tanks.jar mac
```
Comment (pythonmcpi) [inline run.sh:1]: Do we want to keep this script in the pull request? It looks like something used for testing.

Comment (ghostlypi) [reply run.sh]: Good point, however, we do want a way for non-savy users to run the game from the build. As such, I have modified the run command to work more universally on *nix systems.

**Pull request merged**